lighthouse-core/test/config/config-test.js

```diff
@@ -212,10 +211,6 @@ describe('Config', () => {
     }), /audit\(\) method/);
 
     assert.throws(_ => new Config({
-      audits: [basePath + '/missing-category'],
```

Comment:
will you also delete the file this was loading (`lighthouse-core/test/fixtures/invalid-audits/missing-category.js`)